packages/server/postgres/updatePageAccessTable.ts

```diff
@@ -56,17 +56,16 @@ export const updatePageAccessTable = (
         .expression((eb) => eb.selectFrom('nextPageAccess').select(['userId', 'pageId', 'role']))
         .onConflict((oc) =>
           oc
-
             .columns(['userId', 'pageId'])
             .doUpdateSet((eb) => ({
               role: eb.ref('excluded.role')
             }))
             .where(({eb, ref}) => eb('PageAccess.role', 'is distinct from', ref('excluded.role')))
         )
     )
-
-  if (!skipDeleteOld) {
-    return res.with('deleteOld', (qc) =>
+  if (skipDeleteOld) return res.selectNoFrom(sql`1`.as('t')).execute()
```

Comment:
-1 the return value of this function is never used, so why bother with the fake query?